server/src/main/java/org/apache/cassandra/sidecar/cluster/CQLSessionProviderImpl.java

```diff
@@ -70,7 +70,7 @@ public class CQLSessionProviderImpl implements CQLSessionProvider
 {
     private static final Logger logger = LoggerFactory.getLogger(CQLSessionProviderImpl.class);
     private final List<InetSocketAddress> contactPoints;
-    private final int numConnections;
+    private final int numAdditionalConnections;
```

Comment:
changing the name because when I was running tests I had issues, internally this is actually called numAdditionalConnections, so making the change here to reflect what it really signifies